src/test/java/uk/gov/laa/pfla/auth/service/service/MappingTableServiceTest.java

```diff
@@ -0,0 +1,139 @@
+package uk.gov.laa.pfla.auth.service.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.laa.pfla.auth.service.dao.MappingTableDao;
+import uk.gov.laa.pfla.auth.service.exceptions.DatabaseReadException;
+import uk.gov.laa.pfla.auth.service.exceptions.ReportIdNotFoundException;
+import uk.gov.laa.pfla.auth.service.models.MappingTableModel;
+import uk.gov.laa.pfla.auth.service.responses.ReportListEntry;
+import uk.gov.laa.pfla.auth.service.services.MappingTableService;
+
+import java.io.IOException;
```

Comment:
IOException and ArrayList are unused imports

Reply:
removed